index.js

```diff
@@ -82,23 +79,6 @@
 //. }
 //. ```
 //.
```

Comment:
I was keeping those lines in to keep the "good looking" markdown. Not sure whether you cared.